pkg/windows/windows.go

```diff
@@ -857,15 +864,19 @@ func (vm *windows) ensureHNSNetworksAreRemoved() error {
 	// VIP HNS endpoint created by the operator is also deleted when the HNS networks are deleted.
 	for _, network := range []string{BaseOVNKubeOverlayNetwork, OVNKubeOverlayNetwork} {
 		err = wait.Poll(retry.Interval, retry.Timeout, func() (bool, error) {
+			// retry removing network on failure
 			if err := vm.removeHNSNetwork(network); err != nil {
-				return false, errors.Wrapf(err, "error removing %s HNS network", network)
+				vm.log.V(1).Error(err, "error removing %s HNS network", "network", network)
+				return false, nil
 			}
 			if err := vm.Reinitialize(); err != nil {
-				return false, errors.Wrapf(err, "error reinitializing VM after removing %s HNS network", network)
+				vm.log.V(1).Error(err, "error reinitializing VM after removing HNS network", "network", network)
+				return false, nil
```

Comment:
If `vm.Reinitialize()` failed how would any commands over ssh work? So you would have to retry the `vm.Reinitialize()` call or fail on the error.